data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -6,29 +6,21 @@ export KUBECONFIG=/opt/openshift/auth/kubeconfig-loopback
 
 # Wait till the baremetalhosts are populated
 until oc get baremetalhosts -n openshift-machine-api; do
-   echo Waiting for BareMetalHosts to appear...
+   echo Waiting for BareMetalHosts CR to appear...
```

Comment:
CRD?

Reply:
I think CR is right, we're waiting for resources, not definitions?

Reply:
I think the next one is that - it's counting how many resources of that type to see if it's non-zero.
This one is checking if the command fails because there is no such type (because the CRD is not registered yet).
At least, I _think_ that's what's happening.

Reply:
CRD is correct, fixed